CoffeeChat/Controller/DemographicsViewController.swift

```diff
@@ -36,125 +47,109 @@ class DemographicsViewController: UIViewController {
+        // Renders the valid graduation years based on current year.
+        let currentYear = Calendar.current.component(.year, from: Date())
+        for year in currentYear...currentYear+4 {
+            classSearchFields.append("\(year)")
         }
-        hometownSearchBar.setImage(UIImage(), for: .search, state: .normal)
-        hometownSearchBar.layer.cornerRadius = fieldsCornerRadius
-        hometownSearchBar.searchTextPositionAdjustment = UIOffset(horizontal: -6, vertical: 0)
-        hometownSearchBar.showsCancelButton = false
-        view.addSubview(hometownSearchBar)
-
-        pronounsTextField.backgroundColor = .backgroundLightGray
-        pronounsTextField.attributedPlaceholder = NSAttributedString(
-            string: "Pronouns",
-            attributes: [NSAttributedString.Key.foregroundColor: UIColor.textDarkGray]
-        )
-        pronounsTextField.leftView = UIView(frame: CGRect(x: 0, y: 0, width: 12, height: 49))
-        pronounsTextField.leftViewMode = .always
-        pronounsTextField.layer.cornerRadius = fieldsCornerRadius
-        view.addSubview(pronounsTextField)
+
+        classDropdownView = OnboardingSelectDropdownView(delegate: self, placeholder: "Class of...", tableData: classSearchFields, textTemplate: "Class of")
+        classDropdownView.tag = 0 // Set tag to keep track of field selection status.
+        view.addSubview(classDropdownView)
+
+        majorSearchView = OnboardingSearchDropdownView(delegate: self, placeholder: "Major", tableData: majorSearchFields)
+        majorSearchView.tag = 1 // Set tag to keep track of field selection status.
+        view.addSubview(majorSearchView)
+
+        hometownSearchView = OnboardingSearchDropdownView(delegate: self, placeholder: "Hometown", tableData: hometownSearchFields)
+        hometownSearchView.tag = 2 // Set tag to keep track of field selection status.
+        view.addSubview(hometownSearchView)
+
+        pronounsDropdownView = OnboardingSelectDropdownView(delegate: self, placeholder: "Pronouns", tableData: pronounSearchFields, textTemplate: "")
+        pronounsDropdownView.tag = 3 // Set tag to keep track of field selection status.
+        view.addSubview(pronounsDropdownView)
+
+        onboardingSearchViews = [majorSearchView, hometownSearchView]
+        onboardingSelectViews = [classDropdownView, pronounsDropdownView]
 
         nextButton.setTitle("Next", for: .normal)
-        nextButton.setTitleColor(.textBlack, for: .normal)
-        nextButton.backgroundColor = .backgroundLightGray
+        nextButton.setTitleColor(.white, for: .normal)
+        nextButton.titleLabel?.font = ._20CircularStdBold
+        nextButton.backgroundColor = .inactiveGreen
+        nextButton.isEnabled = false
         nextButton.layer.cornerRadius = 26
+        nextButton.layer.shadowColor = .none
+        nextButton.layer.shadowOffset = CGSize(width: 0.0 , height: 0.0)
+        nextButton.layer.shadowOpacity = 0
+        nextButton.layer.shadowRadius = 0
```

Comment:
does this make it so that there is no shadow? im pretty sure there is no shadow by default